packages/manager/modules/pci/src/components/project/billing/hourly-resource-list/translations/Messages_en_GB.json

```diff
@@ -16,5 +16,6 @@
   "cpbc_hourly_floating_ip_title": "Floating IP",
   "cpbc_hourly_gateway_title": "Gateway",
   "cpbc_hourly_ai_deploy_title": "AI Deploy",
-  "cpbc_cold_archive_detail_title": "Cold Archive"
+  "cpbc_cold_archive_detail_title": "Cold Archive",
+  "cpbc_hourly_public_ip_title": "Public IP"
```

Comment:
**nitpick:** Modify the `Messages_fr_FR` not in the `en_GB` locale file.

Reply:
my bad I will correct it